tests/unit/bin/test_build-locales.js

```diff
@@ -0,0 +1,23 @@
+import { getLocaleModulePath } from 'bin/build-locales';
```

Comment:
we could configure webpack/node to create modules and use `@amo` instead of `amo`.

Reply:
(or just `amo`)

Reply:
If this would allow us to stop configuring module paths in four different places then I like this idea! I filed https://github.com/mozilla/addons-frontend/issues/6677